src/mainwindow.cpp

```diff
@@ -474,7 +474,16 @@ MainWindow::MainWindow(Settings &settings
     if (m_PluginContainer.implementInterface<IPluginModPage>(plugin)) { updateModPageMenu(); } });
   connect(&m_PluginContainer, &PluginContainer::pluginDisabled, this, [this](IPlugin* plugin) {
     if (m_PluginContainer.implementInterface<IPluginModPage>(plugin)) { updateModPageMenu(); } });
-
+  connect(&m_PluginContainer, &PluginContainer::pluginRegistered, this, [this](IPlugin* plugin) {
```

Comment:
Can you refactor those?

Reply:
What do you mean? Use the same lambda for both?

Reply:
Maybe move the three function calls to its own function with a name that makes sense? Not sure what the name should be, something like `updateForPluginsChanged()` or `onPluginRegistrationChanged()` or whatever. I just get itchy when I see the same three functions called in two places.